CHANGELOG.md

```diff
@@ -14,8 +14,11 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### New features
 
+<<<<<<< Conflict 1 of 1
++++++++ Contents of side #1
```

Comment:
Nit: Remove the partially commited conflicted resolution.

Reply:
done